easybuild/easyconfigs/m/MOABS/MOABS-1.3.2-foss-2019b.eb

```diff
@@ -0,0 +1,41 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'MakeCp'
+
+name = 'MOABS'
+version = '1.3.2'
+
+homepage = 'https://genomebiology.biomedcentral.com/articles/10.1186/gb-2014-15-2-r38'
+description = """Model based analysis of bisulfite sequencing data"""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+# https://code.google.com/archive/p/moabs/
+source_urls = ['https://s3.amazonaws.com/deqiangsun/software/%(namelower)s/']
+sources = ['%(namelower)s-v%(version)s.src.x86_64_Linux.data.no_database.tar.gz']
+checksums = ['9f4903182b803de381ae517847575d90faf3aed8d96b65999d619ce6b1bf8a4a']
+
+fix_perl_shebang_for = ['bin/*.pl']
```

Comment:
please move this down, below `files_to_copy` (just a matter of style, doesn't affect the actual installation)